libs/sim/node.rs

```diff
@@ -139,6 +173,25 @@ impl Node {
             return Some(self.queue.remove().unwrap());
         }
     }
+
+    // given an RPC, returns the neighbor it should be sent to
+    pub fn route_rpc(&mut self, rpc: &Rpc) -> String {
```

Comment:
Should be consistent. This is also choose_destination.

Reply:
Fixed.